src/resolvers/mutation.ts

```diff
@@ -32,6 +33,10 @@ export async function upsertSavedItem(
   args,
   context: IContext
 ): Promise<SavedItem> {
+  // const serverSpan = tracer.startActiveSpan('server', {
```

Comment:
this is for custom instrumentation that we want to track. we need to export tracer to enable this